web-client/public/js/graph.js

```diff
@@ -726,12 +726,19 @@ var drawGraph = function (nodes, links, positiveWeights, negativeWeights, sheetT
     };
 
     function smartPathEnd (d, w, h) {
-    // If target node is left of the source node make the distance larger
-    // If target node is left of the source node and thicker than 9, make the distance even larger
-        var MINIMUM_DISTANCE_LEFT = d.strokeWidth > 10 ? 18 : 14;
-        var MINIMUM_DISTANCE = d.target.centerX < d.source.newX ? MINIMUM_DISTANCE_LEFT : 8;
     // For arrowheads when target node is to the left of source node
         var LEFT_ADJUSTMENT = 7;
+        var MINIMUM_DISTANCE = 8;
+
+        var targetStartX = d.target.centerX + d.target.textWidth / 2;
+        var currentPointX = (targetStartX - d.target.centerX) / (d.source.newX - d.target.centerX);
+        var currentPointY = (1 - currentPointX) * d.target.centerY + currentPointX * d.source.newY;
+        var nodeHeight = 30;
```

Comment:
Looks like `nodeHeight` is functionally a constant, so can be moved up. The full `nodeHeight` is also not actually used, but is divided by two, so perhaps the constant itself can simply be:
```javascript
    var NODE_HALF_HEIGHT = 30 / 2;
```
(we keep the division here to emphasize that the actual height is 30 and we are dividing by 2)

Reply:
Fixed, thank you.